recipes/sobjectizer/all/conanfile.py

```diff
@@ -47,16 +41,29 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
-    def validate(self):
-        minimal_cpp_standard = "17"
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, minimal_cpp_standard)
-        minimal_version = {
+    def _compiler_support_lut(self):
+        if self.version >= Version("5.8.0"):
+            # Since v5.8.0 requirements to compilers were updated:
+            return {
+                "gcc": "10",
+                "clang": "11",
+                "apple-clang": "13",
+                "Visual Studio": "17"
+            }
+
+        return {
             "gcc": "7",
             "clang": "6",
             "apple-clang": "10",
             "Visual Studio": "15"
         }
+
+    def validate(self):
+        minimal_cpp_standard = "17"
```

Comment:
Is C++17 correct for pre-5.8.0 as well? I would assume it's lower based on the required compiler versions.

Reply:
To my knowledge the answer is - yes.
@eao197, please, confirm.

Reply:
Yes, C++17 is required since v.5.6.0 (it's 2019).